src/sfdxProject.ts

```diff
@@ -39,6 +43,9 @@ export type PackageDir = {
   postInstallUrl?: string;
   releaseNotesUrl?: string;
   uninstallScript?: string;
+  unpackagedMetadata?: {
+    path: string;
+  };
```

Comment:
I wonder if it would be better to have this type live in the schema repo.  I think it should be generated from the schema as well so that any changes to it are automatically reflected in the type.

Reply:
for clarity: you'd want to have some script in `schemas` that rewrites exported TS types based on their json-schema, then sfdx-core would import (and would have to re-export to avoid breaking changes in coreV2) for those types?